src/maintenance/interfaces/IClinicSteward.sol

```diff
@@ -22,6 +38,12 @@ interface IClinicSteward is IRescuableBase, IAccessControl {
   /// @notice The Aave collector
   function COLLECTOR() external view returns (address);
 
+  /// @notice The Aave oracle
+  function ORACLE() external view returns (address);
+
+  /// @notice The rest dollar pull limit
```

Comment:
it's with 8 decimals/oracle precision right? should be in the comment.

again the name is a bit weird `availableBudget` or sth imo would be clearer.